.github/workflows/PrettyTables.yaml

```diff
@@ -0,0 +1,64 @@
+name: PrettyTable
```

Comment:
Let's remove this file, I already have GitHub Actions setup to handle testing, linting, and releasing